src/package-compatibility.js

```diff
@@ -70,6 +71,10 @@ export function testEngine(name: string, range: string, versions: Versions, loos
     return true;
   }
 
+  if (name === 'yarn' && satisfiesWithPreleases(actual, range, looseSemver)) {
```

Comment:
I could have made the check on line 70 use satisifiesWithPrerelease and not bothered with the yarn specific check, but it felt safer to just do yarn for now.